src/components/custom/_signup-new.js

```diff
@@ -62,14 +62,14 @@ const NoteBox = styled.div`
     display: flex;
     flex-direction: row;
     width: 100%;
-    height: 7.7rem;
+    min-height: 7.7rem;
     padding: 8px 8px;
     margin-top: 1.6rem;
     border-radius: 0.4rem;
     background-color: rgba(242, 243, 244, 0.56);
 
     @media ${device.mobileL} {
-        height: 80px;
+        min-height: 80px;
```

Comment:
```suggestion
        min-height: 8rem;
```

Reply:
In deriv.com, for mobile breakpoints, `rem` units have a different ratio in comparison to `px` units. So using `px` unit will be more accurate and is fine